src/NVMLIOGroup.cpp

```diff
@@ -0,0 +1,673 @@
+                                  string_format_double
+                                  }},
+                              {"NVML::TEMPERATURE", {
+                                  "Accelerator temperature in degrees Celsius",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::TOTAL_ENERGY_CONSUMPTION", {
+                                  "Accelerator energy consumption in joules since the driver was loaded",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::sum,
+                                  string_format_double
+                                  }},
+                              {"NVML::PERFORMANCE_STATE", {
+                                  "Accelerator performance state",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::PCIE_RX_THROUGHPUT", {
+                                  "Accelerator PCIE receive throughput in bytes per second over a 20 millisecond period",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::PCIE_TX_THROUGHPUT", {
+                                  "Accelerator PCIE transmit throughput in bytes per second over a 20 millisecond period",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::CPU_ACCELERATOR_ACTIVE_AFFINITIZATION", {
+                                  "Returns the associated accelerator for a given CPU as determined by running processes."
+                                  "\n  If no accelerators map to the CPU then NAN is returned"
+                                  "\n  If multiple accelerators map to the CPU -1 is returned",
+                                  {},
+                                  GEOPM_DOMAIN_CPU,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::UTILIZATION_MEMORY", {
+                                  "Percentage of time the accelerator memory was accessed in the last set of driver samples",
```

Comment:
Fraction rather than percentage would be better.

Reply:
Fixed.  Description change only, signals was already reporting fraction.  UTILIZATION_ACCELERATOR also updated